### DISCUSSION TITLE
Can't manage to make the 'subplot' tool to work.

### DISCUSSION THREAD

**Discussion by antonfrancois** [marcomusy/vedo#514] [Q&A] [answered]
Hello,

I am trying to make a viewer for comparing 3D images side by side. I have several questions but I'll try to separate then in different minimal questions.
Here is the first :  I can't manage to see two volumes in the same time, with shared view (when moving one, it moves the other).
What I have done now, inspired by the [```recosurface```](https://github.com/marcomusy/vedo/blob/master/examples/advanced/recosurface.py) example, have a strange behaviour. When I run the script the first volumes shows, then when I close the window, the second one shows until I re-close the window. 

How to manage to both volumes together ? (I guess it is a small tweak)

```python
import vedo

filename_1 = vedo.dataurl+'embryo.tif'
filename_2 = vedo.dataurl+'vase.vti'

plt = vedo.Plotter(N=2,axes=0)
plt.interactive = True

vol_1 = vedo.Volume(filename_1)
plt.show(vol_1,at=0)

vol_2 = vedo.Volume(filename_2)
plt.show(vol_2,at=1,interactive=1).close()
```

**Comment by marcomusy** [comment, marked as answer]
Than happens because the two objects live in VERY different scales!
the first spans 12000, the second only 100.
Press R after clicking on the renderer to reset the common camera:
```python
import vedo

filename_1 = vedo.dataurl+'embryo.tif'
filename_2 = vedo.dataurl+'vase.vti'

plt = vedo.Plotter(N=2, axes=1)

vol_1 = vedo.Volume(filename_1)
plt.show(vol_1, at=0)

vol_2 = vedo.Volume(filename_2)
plt.show(vol_2, at=1, interactive=True)
```
or can use `sharecam=False` to have independent cams.

**Reply by antonfrancois** [reply]
Thanks ! That was the problem indeed.
